test/ecto/repo/autogenerate_test.exs

```diff
@@ -85,6 +85,7 @@ defmodule Ecto.Repo.AutogenerateTest do
     default = TestRepo.insert!(%Company{})
     assert %NaiveDateTime{microsecond: {0, 0}} = default.inserted_at
     assert %NaiveDateTime{microsecond: {0, 0}} = default.updated_at
+    assert default.inserted_at == default.updated_at
```

Comment:
I would either include this assertion in all the tests, or [here](https://github.com/elixir-ecto/ecto/blob/f676c115210894287a89a8f43a7dd9b4ef15ba7a/test/ecto/repo/autogenerate_test.exs#L170).

I think this is a flaky test with the current implementation.